src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -175,6 +175,26 @@ void Goo()
             await TestInRegularAndScriptAsync(code, expected, index: 1);
         }
 
+        [WorkItem(21747, "https://github.com/dotnet/roslyn/issues/21747")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
+        public async Task TestTriviaFieldFix1()
+        {
+            var code =
+@"class C
+{
+    int i = (/* CommentLeading */ [|1 + 1|] /* CommentTrailing */) + (1 + 1);
```

Comment:
:bulb: If you make the selected expression `1 + /*CommentEmbedded*/ 1` it will show that the embedded comment is moved while the leading/trailing stay at the reference site.

Reply:
Test case added and it behaves as expected.